.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-3.2.2
```

Comment:
Since you are using asdf, there should be a `.tool-versions` file instead of a `.ruby-version` file.